src/components/AudioPlayer/AudioPlayer.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import { action } from "@storybook/addon-actions";
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs";
+import { withDesign } from "storybook-addon-designs";
+
+import AudioPlayer from "./AudioPlayer";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("AudioPlayer")}
+  component={AudioPlayer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=36835%3A25338",
+    },
+    jest: ["AudioPlayer.test.tsx"],
+  }}
+  argTypes={{
+    audioType: {},
+    className: { control: false },
+    id: { control: false },
+  }}
+/>
+
+# Audio Player
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.1.0`    |
+| Latest            | `1.1.0`    |
```

Comment:
I think this might be `1.2.0`.

Reply:
Yes, the next point release will be `1.2.0`, so let’s use that here.  I would like to include this component in that release.

Reply:
Updated. Thanks!